resources/js/confirm-delete.js

```diff
@@ -1,10 +1,45 @@
+// Show confirm dialog before submitting any spoofed DELETE method form
 document.body.addEventListener("submit", function(event) {
-  if (event.target.querySelector('input[name="_method"][value="delete"], input[name="_method"][value="DELETE"]')) {
+  let method;
+  let deleteButton;
+
+  // Take last spoofed method using FormData if available
+  if (FormData instanceof Object) {
+    let formData = new FormData(event.target);
+    if (typeof formData.getAll == "function") {
+      method = (
+        formData.getAll("_method").pop() || event.target.method
+      ).toUpperCase();
+    }
+  }
+
+  [...event.target.elements].forEach(function(input) {
```

Comment:
Sorry for being slow here, finally understand what's going on here. :-) The spread operator is turning HTMLFormControlsCollection into an Array so you're able to use the forEach method, neat! The spread operator is quite a powerful thing!